utilities/data_structure_test.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "rocksdb/utilities/data_structure.h"
+
+#include "db/db_test_util.h"
+#include "rocksdb/types.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+// Unit test for SmallEnumSet
+class SmallEnumSetTest : public testing::Test {
+ public:
+  SmallEnumSetTest() {}
+  ~SmallEnumSetTest() {}
+};
+
+TEST_F(SmallEnumSetTest, SmallSetTest) {
+  FileTypeSet fs;
+  ASSERT_TRUE(fs.Add(FileType::kIdentityFile));
+  ASSERT_TRUE(fs.Add(FileType::kInfoLogFile));
```

Comment:
Need to test a case of Add returning false. (Otherwise, fine 👍 )

Recommended: combine with an existing extremely short-running test binary like slice_test or heap_test.

And this seems more like `util/` (useful outside of RocksDB) than `utilities/` (accessories specific to RocksDB)

Reply:
Sure, will make the change